docs/overview_usecases.rst

```diff
@@ -15,8 +15,8 @@ that can perform calculations. libEnsemble's work is governed by three routines:
 * :ref:`alloc_f<api_alloc_f>`: Decides whether ``sim_f`` or ``gen_f`` should be called (and with what input/resources) as workers become available
 
 Example ``gen_f``, ``sim_f``, ``alloc_f``, and calling scripts can be found in
-the ``examples/`` directory. In order to enable portability, a :doc:`job_controller<job_controller/overview>`
-interface is supplied for users to launch and monitor external scripts in their
+the ``examples/`` directory. In order to enable portability, an :doc:`executor<executor/overview>`
+interface is supplied for users to execute and monitor external scripts in their
```

Comment:
Should this "execute" be "submit"?

Reply:
For this context, I think I prefer "execute" since that is the eventual action of the executor and best describes for those reading the overview that libEnsemble can run their code.

Thoughts?

Reply:
I think 'scripts' should be 'applications'.

Reply:
"Applications" is a good word for describing to users and within the docs exactly what the executor is responsible for. I'm checking myself, but if anyone finds somewhere where "jobs", "tasks", or "scripts" should be "applications", please let me know.